### PR TITLE
kafka: fix audit error

### PR DESCRIPTION
**Pull request by OndroNR**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

---

Fixes following audit error:

```
brew audit --strict kafka
kafka:
  * Use `system "echo", "\"` instead of `system "echo \"`
Error: 1 problem in 1 formula
```

### CODE REVIEW

File: Formula/kafka.rb [hunk 1]
```diff
@@ -104,7 +104,7 @@ def plist; <<-EOS.undent
       sleep 30
 
       system "#{bin}/kafka-topics --zookeeper localhost:2181 --create --if-not-exists --replication-factor 1 --partitions 1 --topic test > #{testpath}/kafka/demo.out 2>/dev/null"
-      system "echo \"test message\" | #{bin}/kafka-console-producer --broker-list localhost:9092 --topic test 2>/dev/null"
+      system "#{bin}/kafka-console-producer --broker-list localhost:9092 --topic test 2>/dev/null <<< 'test message'"
```
Comment (bfontaine) [inline Formula/kafka.rb]: Oops sorry it should be `pipe_output`.

Comment (OndroNR) [inline Formula/kafka.rb]: Works now, thanks

**Comment by dunn** [comment]
The test is failing on Sierra:
```
==> /usr/local/Cellar/kafka/0.10.0.1/bin/kafka-topics --zookeeper localhost:2181 --create --if-not-exists --replication-factor 1 --partitions 1 --topic test > /tmp/kafka-test-20161030-78740-1t42q38/kafka/demo.out 2>/dev/null
==> /usr/local/Cellar/kafka/0.10.0.1/bin/kafka-console-producer --broker-list localhost:9092 --topic test 2>/dev/null
==> /usr/local/Cellar/kafka/0.10.0.1/bin/kafka-console-consumer --zookeeper localhost:2181 --topic test --from-beginning --max-messages 1 >> /tmp/kafka-test-20161030-78740-1t42q38/kafka/demo.out 2>/dev/null
==> /usr/local/Cellar/kafka/0.10.0.1/bin/kafka-server-stop
==> /usr/local/Cellar/kafka/0.10.0.1/bin/zookeeper-server-stop
Error: kafka: failed
execution expired

**Comment by OndroNR** [comment]
I don't have Sierra installed yet. Works on El Capitan. Does kafka test on master branch work?

**Comment by MikeMcQuaid** [comment]
@OndroNR You can test that with `brew test kafka` on the `master` branch. I've restarted CI in case it's a one-off error.

**Comment by dunn** [comment]
Merged as f41d91bc8a2e4e90a0166c8f5ae24b0f3c642e0b, thanks!

**Pull request closed without merging**